MiFloreria/Sources/LoginFeature/Home/HomeViewController.swift

```diff
@@ -87,7 +87,7 @@ class HomeViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = UIColor.dsBackground
+        view.backgroundColor = .clear
```

Comment:
Añade el color correcto.
```suggestion
        view.backgroundColor = UIColor(named:"DSBackground")
```